game_patch/misc/vpackfile.cpp

```diff
@@ -61,6 +64,26 @@ static bool is_mod_file_in_whitelist(const char* Filename)
 
 #endif // MOD_FILE_WHITELIST
 
+// allow list of table files that can't be used for cheating, but for which modding in clientside mods has utility
+// Examples include translation packs (translated strings, endgame, etc.) and custom HUD mods that change coords of HUD elements
+const char* tbl_mod_allow_list[] = {
```

Comment:
Should be `std::array`.

Reply:
I've matched the standard set by rafalh's existing code for this (from `MOD_FILE_WHITELIST`). I have no problem changing it if it makes sense, but I'm going to leave it as-is unless he decides it makes sense to change both to std:array.

Reply:
Modern C++ is very memory unsafe. C arrays and C strings are _even more_ memory unsafe.  Every C++ guideline is going to say even if it is a hassle where feasible use modern C++. In this case it is `std::array` and `std::string_view` and `std::string` which he does use throughout his code.